src/tutorials/pet.md

```diff
@@ -19,13 +19,13 @@ and others use proprietary formats for example ECAT format (.v).
 In order to facilitate easy testing of data conversion across different PET file formats
 the [OpenNeuroPET project][OpenNeuroPET project] has compiled a bunch of phantom data
 from different scanner types and is distributing two examples
-[here](https://drive.google.com/file/d/10S0H7HAnMmxHNpZLlifR14ykIuiXcBAD/view?usp=sharing).
```

Comment:
Can I let you fix them on the bids website too? (the tutorials have been migrated)

https://github.com/bids-standard/bids-website/blob/522d8fb5288cb569d5c2013f505e1c2ab5c73a23/docs/getting_started/tutorials/conversion/pet.md?plain=1#L22

Reply:
Yes, this is where I got into trouble fixing this earlier; making an update-broken-links PR on the bids-website to main branch too.